polysemy-zoo.cabal

```diff
@@ -7,7 +7,7 @@ cabal-version: 1.12
 -- hash: 21ba8ceb18ae6184f65237f20847dd9e3f23ad2838d46fc1cd6af719dac66eb3
 
 name:           polysemy-zoo
-version:        0.7.0.2
+version:        0.8.0.0
```

Comment:
This version bump doesn't have a corresponding change in the `package.yaml`. I don't see any reason to bump the version quite yet, so if you've changed the `package.yaml` locally, please return the version number to 0.7.0.2, rerun `stack build`, and push up the `.cabal` field (properly building it matters for the hash).
Alternatively, if you don't use `stack`, I could do it.

Reply:
Actually, thinking about it, 9.2 compatibility is important enough to release this change ASAP, so I was thinking to cut a release tomorrow. So your choice -- either bump the version number in `package.yaml` or not. Either way, `stack build` will need to be run for the hash (whether it be you or I who does it).

Reply:
I don't use stack and I'm not quite sure what you're asking here...

Reply:
I'll do it then, tomorrow. Now I'm going to sleep.